tasks/pkg-debian/install-apm-inject.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Set APM injection languages and versions for Debian
+  set_fact:
+    agent_dd_apm_install_pkgs: "{{ (agent_dd_apm_install_pkgs |
+      default([], true)) + ['datadog-apm-library-' + (item | regex_replace('[:]', '='))] }}"
+  loop: "{{ pre_processed_datadog_apm_libraries }}"
+
+- name: Install APM Inject Library
```

Comment:
Yes, I agree. However due to the nature of having a mixture of both versioned, and "latest" packages, it makes more sense to do it this way since ansible needs the `state: latest` for latest, and `state: present` for versioned packages. I realize this is less than optimal, but really its max 5 calls, so it seems like it makes sense. Plus it lets us handle each call individually in the case of an error.